package.json

```diff
@@ -7,14 +7,17 @@
     "build": "tsc && vite build",
     "build:local": "yarn build",
     "check-types": "tsc -p tsconfig.json --noEmit",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run",
+    "cy:test": "cypress run --spec",
```

Comment:
Since `--spec` needs an argument, it feels more transparent to just use `yarn cy:run --spec <spec>` instead of having a separate command
```suggestion
```